src/app/components/Byline/index.tsx

```diff
@@ -21,154 +21,163 @@ const Byline = ({
-                size="brevier"
-                fontVariant="sansRegularItalic"
-              >
-                {`${reportingFromTranslated} `}{' '}
-              </Text>
-              <Text
-                css={BylineCss.locationText}
-                size="brevier"
-                fontVariant="sansRegular"
-              >
-                {location}
-              </Text>
-            </span>
-          </li>
-        ) : null}
-        {children ? (
-          <li css={BylineCss.timestampLineBreak}>{children}</li>
-        ) : null}
-      </ul>
-    </section>
+              </li>
+              <li>
+                <span role="text">
+                  <VisuallyHiddenText>{`${role}, `} </VisuallyHiddenText>
+                  <Text
+                    css={BylineCss.jobRole}
+                    fontVariant="sansBold"
+                    size="brevier"
+                  >
+                    {jobRole}
+                  </Text>
+                </span>
+              </li>
+              {twitterLink ? (
+                <li>
+                  <a
+                    css={[BylineCss.link, BylineCss.twitterLink]}
+                    className="focusIndicatorReducedWidth"
+                    href={twitterLink}
+                    aria-labelledby="byline-twitter-link"
+                  >
+                    <span role="text" id="byline-twitter-link">
+                      <VisuallyHiddenText lang="en-GB">{`X, `}</VisuallyHiddenText>
+                      <Text
+                        className="byline__link-text"
+                        css={BylineCss.twitterText}
+                        size="brevier"
+                        fontVariant="sansBold"
+                      >{`@${twitterText}`}</Text>
+                      {isRtl ? (
+                        <LeftChevron css={BylineCss.twitterChevron} />
+                      ) : (
+                        <RightChevron css={BylineCss.twitterChevron} />
+                      )}
+                    </span>
+                  </a>
+                </li>
+              ) : null}
+              {location ? (
+                <li>
+                  <span role="text" css={BylineCss.location}>
+                    <Text
+                      css={BylineCss.reportingFromText}
+                      size="brevier"
+                      fontVariant="sansRegularItalic"
+                    >
+                      {`${reportingFrom} `}{' '}
+                    </Text>
+                    <Text
+                      css={BylineCss.locationText}
+                      size="brevier"
+                      fontVariant="sansRegular"
+                    >
+                      {location}
+                    </Text>
+                  </span>
+                </li>
+              ) : null}
+            </ul>
+          );
+        });
+
+  return (
+    contributors?.[0] && (
+      <section role="region" aria-labelledby="article-byline">
```

Comment:
Shall we do an mini / dev a11y swarm on this (focus on screen readers) , given that we have moved this code around? 

We'll need an example with 1 and multiple contributors to make sure it all makes sense.